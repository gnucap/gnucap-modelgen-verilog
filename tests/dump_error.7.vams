
// BUG // suboptimal error message
module test()

	analog begin
		$strobetypo(0);
	end

endmodule
