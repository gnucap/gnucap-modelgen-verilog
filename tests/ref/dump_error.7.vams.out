... st()

	analog begin
		$strobetypo(0);
	end

endmodule

                                        ^ ? bad discipline
